opennms-base-assembly/src/main/filtered/etc/events/A10.AX.events.xml

```diff
@@ -108,8 +108,10 @@
       <descr>Indicates that an SLB virtual service port has come up.</descr>
       <logmsg dest="logndisplay">&lt;p> A10 trap received Description=%parm[#1]% VirtualServer=%parm[#2]%
             Port=%parm[#3]% Type=%parm[#4]%&lt;/p></logmsg>
-      <severity>Normal</severity>
-      <alarm-data reduction-key="%uei%:%parm[#2]%:%parm[#3]%:%parm[#4]%" alarm-type="2" clear-key="uei.opennms.org/traps/A10/axVirtualServerPortDown:%parm[#2]%:%parm[#3]%:%parm[#4]%" auto-clean="false"/>
+      <severity>Cleared</severity>
+      <alarm-data alarm-type="2" reduction-key="uei.opennms.org/traps/A10/axVirtualServerPortDown:%parm[#2]%:%parm[#3]%:%parm[#4]%" auto-clean="false">
+        <update-field field-name="severity" update-on-reduction="true"/>
```

Comment:
What do you think about updating the severity on reduction by default? Would save having to set it for all of these...